bin/alluxio-start.sh

```diff
@@ -137,6 +141,16 @@ stop() {
   ${BIN}/alluxio-stop.sh $1
 }
 
+start_logserver() {
+    echo "Starting logserver @ $(hostname -f)."
+    (nohup "${JAVA}" -cp ${CLASSPATH} \
+     ${ALLUXIO_LOGSERVER_JAVA_OPTS} \
+     alluxio.logserver.AlluxioLogServer "${ALLUXIO_LOGSERVER_BASE_LOGS_DIR}" > ${ALLUXIO_LOGS_DIR}/logserver.out 2>&1) &
+    # Wait for 1s before starting other Alluxio servers, otherwise may cause race condition
```

Comment:
Is the issue that the initial logs get lost, or that the remote socket appender will give up forever if it fails to connect to the log server at startup?